src/components/ChatList/ChannelListHeaderItem.tsx

```diff
@@ -26,14 +27,14 @@ const ChannelListHeaderItem = (props: ChannelListTabItemProps) => {
       paddingRight: dimen.normalizeDimen(30),
       justifyContent: 'center',
       alignItems: 'center',
-      backgroundColor: colors.white,
+      backgroundColor: COLORS.white,
       borderBottomWidth: 2,
-      borderBottomColor: type === SIGNED ? colors.darkBlue : colors.anon_primary
+      borderBottomColor: type === 'SIGNED' ? COLORS.blue : COLORS.holyTosca
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values have been updated to use the new `COLORS` constants. This change improves maintainability by centralizing color definitions in one place, making it easier to update colors consistently across the application in the future.
```diff
- backgroundColor: colors.white,
- borderBottomColor: type === SIGNED ? colors.darkBlue : colors.anon_primary
+ backgroundColor: COLORS.white,
+ borderBottomColor: type === 'SIGNED' ? COLORS.blue : COLORS.holyTosca
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->